docs/source/development.md

```diff
@@ -76,5 +81,8 @@ DB_PORT=$DB_PORT \
 DB_DATABASE=$DB_DATABASE \
 DB_USER=$DB_USER \
 DB_PASSWORD=$DB_PASSWORD \
-python3 manage.py test
+coverage run --source webservice/ --omit */__init__.py,*/urls.py,*/apps.py,*/webservice/tests/* ./manage.py test
```

Comment:
Similarity, I get the following error when I try to run this in the `webserver/pkgpkr/` folder.

```
% coverage run --source webservice/ --omit */__init__.py,*/urls.py,*/apps.py,*/webservice/tests/* ./manage.py test
zsh: no matches found: */__init__.py,*/urls.py,*/apps.py,*/webservice/tests/*
```